### PR TITLE
feat(scenarios): add help text and tooltips to scenario form fields

### LINKED ISSUE #1125: OPSCEN-30: Scenario Authoring Help Text
## Goal

Improve in-context guidance for Scenario authoring by adding meaningful placeholder text and help tooltips to the Situation, Labels, and Criteria fields.

## Design

User feedback indicates the editor needs better guidance. The excellent content from the authoring methodology should be surfaced in the UI.

### Situation Field

**Placeholder text:**
```
Who is the user? What's their context?
What are they trying to accomplish? What emotional state are they in?

Example: A frustrated premium subscriber who was charged twice for the same 
order. They've been loyal for 3 years and want an immediate refund.
```

**Help tooltip (ℹ️):**
> A good situation gives enough context for the simulator to push the agent realistically. Don't describe the "happy path" — describe the scenario that keeps you up at night.

### Labels Field

**Help tooltip (ℹ️):**
> Use labels to categorize scenarios for filtering: #critical, #billing, #regression, #edge-case, etc.

### Criteria Field

**Placeholder (new criterion input):**
```
What must the agent do or NOT do? Write in plain language.
```

**Help tooltip (ℹ️):**
> Criteria are behavioral guardrails. Think about:
> - **Behavioral:** "Must remain empathetic"
> - **Policy:** "Must NOT offer X when Y"
> - **Functional:** "Must suggest Z if condition"
> - **Execution:** "Must confirm email before lookup"

## Tasks

| ID | Task | Size |
|:---|:-----|:-----|
| SCEN-30a | Add placeholder text to Situation textarea | S |
| SCEN-30b | Add help tooltip component to Situation field | S |
| SCEN-30c | Add help tooltip to Labels field | S |
| SCEN-30d | Add placeholder and help tooltip to Criteria input | S |

**Estimate:** ~0.5-1 day

## Checklist

- [ ] Situation field placeholder text
- [ ] Situation field help tooltip with info icon
- [ ] Labels field help tooltip
- [ ] Criteria input placeholder text
- [ ] Criteria field help tooltip
- [ ] Consistent tooltip styling across fields

## Files to Modify

```
src/components/scenarios/
├── ScenarioForm.tsx          # Add placeholders and help tooltips
└── (or relevant form components)
```

## Reference

- Planning repo: `docs/scenario/authoring-methodology.md` — Source of guidance language
- Planning repo: `docs/scenario/wireframes/02-editor.md` — Current editor wireframe

### PR DESCRIPTION
**Pull request by drewdrewthis**
## Summary
- Add helper text under section headers for Situation and Criteria fields explaining how to write effective content
- Add helper text under Labels field explaining how to use labels for filtering
- Add italic styling to placeholder text to distinguish from filled-in values
- Simplify placeholder examples (short "e.g." format)

Closes #1125

## Test plan
- [ ] Verify helper text appears under each section header (Situation, Criteria) and Labels field
- [ ] Verify placeholder text is italicized and visually distinct from actual content
- [ ] Verify form still functions correctly (add/remove criteria, labels, submit)

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: langwatch/src/components/scenarios/ScenarioForm.tsx [hunk 1]
```diff
@@ -106,48 +106,63 @@ export function ScenarioForm({ defaultValues, formRef }: ScenarioFormProps) {
...
       </VStack>
 
       {/* SITUATION Section */}
       <VStack align="stretch" gap={3}>
-        <SectionHeader>Situation</SectionHeader>
+        <VStack align="stretch" gap={1}>
+          <SectionHeader>Situation</SectionHeader>
+          <Text fontSize="13px" color="gray.500">
+            Describe the user, their context, and what they're trying to
+            accomplish. Don't describe the "happy path" — describe the scenario
```
Comment (rogeriochaves) [inline langwatch/src/components/scenarios/ScenarioForm.tsx]: idk man, I think having the happy path is nice to work as a smoke test, I'd prefer to be more like, "Think about a critical path or a complex edge case."

File: langwatch/src/components/scenarios/ScenarioForm.tsx [hunk 2]
```diff
@@ -106,48 +106,63 @@ export function ScenarioForm({ defaultValues, formRef }: ScenarioFormProps) {
...
         </Field.Root>
       </VStack>
 
       {/* CRITERIA Section */}
       <VStack align="stretch" gap={3}>
-        <SectionHeader>Criteria</SectionHeader>
+        <VStack align="stretch" gap={1}>
+          <SectionHeader>Criteria</SectionHeader>
+          <Text fontSize="13px" color="gray.500">
+            What must the agent do or NOT do? e.g. "Must remain empathetic",
```
Comment (rogeriochaves) [inline langwatch/src/components/scenarios/ScenarioForm.tsx]: should we caps also the first DO?

`DO or NOT DO?`
